client/src/components/Loading/index.js

```diff
@@ -12,20 +12,29 @@ const useStyles = makeStyles((theme) => ({
     marginTop: '14%',
     color: theme.palette.primary,
   },
+  divHeight: {
+    height: '200px',
+  },
 }));
 
 const LoadingProgressBar = (props) => {
-  const { circleSize } = props;
+  const { height200px, circleSize } = props;
   const classes = useStyles();
 
   return (
-    <div className={classes.root}>
+    <div className={`${classes.root} ${height200px ? classes.divHeight : ''}`}>
```

Comment:
This is minor but I'd suggest a slightly less specific prop name than `height200px`.  For example, `isFixedHeight` or `isTall`.  Then you won't have specific css-related prop names appearing in other files which use this component (and also you wouldn't have an incorrect name if you ever decided to change the actual pixel height to, say, 250px)